pkg/cmd/completion.go

```diff
@@ -12,6 +16,17 @@ type completionCmd struct {
 	shell string
 }
 
+var installDirectory = map[string]map[string]string{
+	"darwin": map[string]string{
+		"zsh":  "/usr/local/share/zsh/site-functions",
+		"bash": "/usr/local/etc/bash_completion.d",
+	},
+	"linux": map[string]string{
+		"zsh":  "/usr/share/zsh/functions/Completion/Linux",
+		"bash": "/usr/share/bash-completion/completions",
```

Comment:
I'm not sure we should be making assumptions about directories in the CLI. Apparently on Linux `/etc/bash_completion.d/` can also be used.